packages/pluggableWidgets/datagrid-number-filter-web/src/DatagridNumberFilter.editorConfig.ts

```diff
@@ -34,32 +34,36 @@ export const getPreview = (values: DatagridNumberFilterPreviewProps): StructureP
                 columnSize: "grow",
                 backgroundColor: "#FFFFFF",
                 children: [
-                    {
-                        type: "Container",
-                        padding: 4,
-                        grow: 0,
-                        children: [
-                            {
-                                type: "Image",
-                                document: getSvgContent(values.defaultFilter)
-                            } as ImageProps
-                        ]
-                    } as ContainerProps,
-                    {
-                        type: "Container",
-                        borders: true,
-                        borderWidth: 0.5,
-                        grow: 0
-                    } as ContainerProps,
+                    ...(values.adjustable
+                        ? [
+                              {
+                                  type: "Container",
+                                  padding: 2,
+                                  grow: 0,
+                                  children: [
+                                      {
+                                          type: "Image",
+                                          document: getSvgContent(values.defaultFilter)
+                                      } as ImageProps
+                                  ]
+                              } as ContainerProps,
+                              {
+                                  type: "Container",
+                                  borders: true,
+                                  borderWidth: 0.5,
+                                  grow: 0
+                              } as ContainerProps
+                          ]
+                        : []),
```

Comment:
Similar comment for this to extract into const to make it more readable

Reply:
Fixed